x-pack/plugins/security_solution/common/detection_engine/schemas/response/rules_schema.mocks.ts

```diff
@@ -110,3 +110,11 @@ export const getThreatMatchingSchemaMock = (anchorDate: string = ANCHOR_DATE): R
     ],
   };
 };
+
+export const getRulesEqlSchemaMock = (anchorDate: string = ANCHOR_DATE): RulesSchema => {
+  const baseRule = getRulesSchemaMock(anchorDate);
+  delete baseRule.language;
+  baseRule.type = 'eql';
+  baseRule.query = 'process where true';
+  return baseRule;
+};
```

Comment:
Another way I have seen these written as has been like so:

```ts
export const getRulesEqlSchemaMock = (anchorDate: string = ANCHOR_DATE): RulesSchema => {
  const { language, ...noLanguage } = getRulesSchemaMock(anchorDate);
  return {
    ...noLanguage,
    type: 'eql',
    query: 'process where true',
  };
};
````

No changes asked, just pointing out another way if you wanted to avoid the delete and the assignments.